pkgs/applications/misc/girara/default.nix

```diff
@@ -19,19 +19,16 @@
 
 stdenv.mkDerivation rec {
   pname = "girara";
-  version = "0.4.3";
+  version = "0.4.5";
 
   outputs = [
     "out"
     "dev"
   ];
 
-  src = fetchFromGitLab {
```

Comment:
What's the problem with using a git checkout rather than upstream's tarball. I think we generally prefer the former due to incidents like xz.
You just have to switch to `fetchFromGitHub`: https://github.com/pwmt/girara